LoadInterests.py

```diff
@@ -0,0 +1,13 @@
+import csv
+
+def main(interests):
+    # code to use the interests in the main function
+    pass
+
+with open('interests.csv', 'r') as file:
+    csv_reader = csv.reader(file)
+    interests = []
+    for row in csv_reader:
+        interests.append(row[0])
+    only_interests = interests_with_header[1:]
+    main(interests)
```

Comment:
this should be a function, maybe the name `load_interest()` and it should just return the loaded interests

Reply:
this function should be called in src/main.py here https://github.com/Srinika99/Blog-sites-scraping/blob/2216d50bc341e8a37d5d846d5571b7d6cca423af/src/main.py#L6-L7